README.md

```diff
@@ -48,18 +50,43 @@ The project uses `pre-commit` for identifying and fixing simple issues before yo
 
 ### Installation
 
+- Install the packages
+
 ```
 npm install
+```
+
+- Install `pre-commit`
+
+```
 pre-commit install
 ```
 
-### Compiles and hot-reloads for development
+- Copy `.env.example` to `.env.local` and set the appropriate values of the environment variables. The list of all environment variables along with their meanings can be found in [ENV.md](./docs/ENV.md)
+
+### Compile and setup hot-reloading for development
 
 ```
 npm run serve
 ```
 
-### Compiles and minifies for production
+### Compile and minify for deployment
+
+#### Staging
+
+- Copy `.env.example` to `.env.staging.local` and set the appropriate values of the environment variables.
```

Comment:
- Rename `.env.staging.local` to `.env.staging` and `.env.production.local` to `.env.production`
- Add both these to .gitignore